ext/node/polyfills/02_init.js

```diff
@@ -14,33 +14,27 @@ function initialize(
   usesLocalNodeModulesDir,
   argv0,
 ) {
-  if (initialized) {
-    throw Error("Node runtime already initialized");
-  }
-  initialized = true;
-  if (usesLocalNodeModulesDir) {
-    requireImpl.setUsesLocalNodeModulesDir();
+  // No arguments passed when warming up snapshot
+  if (arguments.length > 0) {
```

Comment:
**nitpick:** Accessing arguments object tends to make engines sad. Consider just checking if both of the args are undefined. I presume we never call with explicit undefined arguments after all.

Reply:
I think this may have changed in recent years -- we ran some experiments in deno_core and surprisingly I was not able to measure much difference, even on hot methods that should be fully JITted.

Reply:
I switched to using an explicit warmup param with a default.